.gitignore

```diff
@@ -8,3 +8,8 @@
 # See the `bun.lock` target in `Makefile` for more details.
 /bun.lock
 /bun.lockb
+
+# These would be more appropriate in individual user's `.git/info/exclude`
+# files, but must be here to work around <https://github.com/biomejs/biome/issues/4822>.
+/.direnv
+/.envrc
```

Comment:
Since this is specifically a Biome issue, what would you think of adding them just to `biome.json`?

Reply:
> I think Biome _might_ also support `biome.jsonc` paths by default by now?

Ah, yeah: https://github.com/biomejs/biome/issues/1259

(I think I paged that out of my memory as quickly as possible because there are only so many semantically gnarly things I'm willing to keep in my head.)